docker/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM elasticsearch:latest
+RUN apt-get update && apt-get install -y git
+RUN curl -sL https://deb.nodesource.com/setup_7.x | bash -
+RUN apt-get install -y nodejs
+RUN git clone https://github.com/MusicConnectionMachine/VisualizationG5.git
```

Comment:
You don't need to clone things :) You can simply `COPY` content from the underlying folder/repository ( Eg.: https://github.com/sacdallago/cellmap/blob/develop/Dockerfile#L8 )

This will avoid: latency, network errors (SSH key errors, for example)